src/test/java/org/broadinstitute/hellbender/utils/io/IOUtilsUnitTest.java

```diff
@@ -166,6 +166,21 @@ private void innerTestGetPath(String s) throws IOException {
         Assert.assertTrue(size>0);
     }
 
+    @DataProvider
+    public Object[][] absoluteNames() {
+        return new Object[][] {
```

Comment:
Can you also include a test case with a non-absolute path?

Reply:
Done